srtcore/core.cpp

```diff
@@ -80,13 +80,16 @@ modified by
 
 using namespace std;
 
+namespace logging
```

Comment:
Please use library-specific namespace for the globals, like srt_logging or srt_logging_internal